samples/net/sockets/echo_server/boards/qemu_cortex_a53-e1000.overlay

```diff
@@ -0,0 +1,16 @@
+&pcie {
+	eth0: eth@0 {
+		compatible = "intel,e1000";
+
+		vendor-id = <0x8086>;
+		device-id = <0x100e>;
+
+		interrupts = <2>;
```

Comment:
I'm not quite sure why this works with `2` instead of `1`.
QEMU seems to specify INTx pin 1 in PCI registers, but then uses pin 2...

Reply:
From what I can tell, this is due to PCI-to-PCI bridge INTx "swizzling".
Where the INTx pin is offset by its PCI slot number (bdf function >> 3).
The slot in this case is `1` so INTA# becomes INTB#

(Section 9.1: https://cds.cern.ch/record/551427/files/cer-2308933.pdf)

![image](https://github.com/zephyrproject-rtos/zephyr/assets/124088381/caa59192-b5a1-4361-86ba-8ba648ea06ea)